documentation/book/ref-metrics-config-files.adoc

```diff
@@ -0,0 +1,39 @@
+
+// This assembly is included in the following assemblies:
+//
+// assembly-metrics-setup.adoc
+[id='ref-metrics-config-files-{context}']
+
+= Example Metrics files
+
+[source]
+--
+metrics
+├── grafana-install
+│   ├── grafana.yaml <1>
+├── grafana-dashboards <2>
+│   ├── strimzi-kafka-connect.json
+│   ├── strimzi-kafka.json
+│   └── strimzi-zookeeper.json
+├── kafka-connect-metrics.yaml <3>
+├── kafka-metrics.yaml <4>
+├── prometheus-additional-properties
+│   └── prometheus-additional.yaml <5>
+├── prometheus-alertmanager-config
+│   └── alert-manager-config.yaml <6>
+└── prometheus-install
+    ├── alert-manager.yaml <7>
+    ├── prometheus-rules.yaml <8>
+    ├── prometheus.yaml <9>
+    └── strimzi-service-monitor.yaml <10>
+--
+<1> Grafana docker image
```

Comment:
I do not think this is the best description :(
What about `Installation file for Grafana image`?